source/content/supported-drupal.md

```diff
@@ -40,16 +40,84 @@ terminus site:create <site> <label> drupal8
+Drupal 7 will reach it's end of life on January 5, 2025. Pantheon has partnered with Tag1 Consulting to offer Long-Term Support for Drupal 7 through **January 5, 2027**.
+To learn more about this partnership, see [related blog post](https://pantheon.io/blog/pantheon-and-tag1-support-drupal-7-websites).
 
-#### What's included
-* Security and compatibility updates to Drupal core and core dependencies from Tag1 Consulting by updating their site running on the [Pantheon Drupal 7 Upstream](https://github.com/pantheon-systems/drops-7) via [the Pantheon Dashboard](core-updates#apply-upstream-updates-via-the-site-dashboard) or [Terminus command line tool](/core-updates#apply-upstream-updates-via-terminus).
-* Security patches and compatibility updates to community-contributed modules that power their site via Tag1’s D7 Extended Support (D7ES) module, which will be included in the Upstream.
-* Continued support for Drupal 7-compatible runtime environments on the Pantheon Platform, including PHP, MySQL and other prerequisites.
 
-To learn more about this partnership, see related blog post: [Pantheon and Tag1 Consulting Partner to Provide Long-Term Support for Drupal 7 Websites ](https://pantheon.io/blog/pantheon-and-tag1-support-drupal-7-websites).
+#### D7 Core LTS and Platform Compatibility 
+This includes security and compatibility updates to Drupal core and core dependencies from Tag1 Consulting, which are distributed by the Pantheon Drupal 7 Upstream and made available as a one-click updates via [the Pantheon Dashboard](core-updates#apply-upstream-updates-via-the-site-dashboard) or [Terminus command line tool](/core-updates#apply-upstream-updates-via-terminus).
+
+Also included, is continued support for Drupal 7-compatible runtime environments on the Pantheon Platform, including PHP, MySQL and other prerequisites.
+
+##### Requirements
+* Available to all site plans and workspace tiers.
+* Sites must use the [Pantheon Drupal 7 Upstream](https://github.com/pantheon-systems/drops-7).
+  * Or a Custom Upstream based on the Pantheon Drupal 7 Upstream with the [latest updates pulled in](/guides/custom-upstream/create-custom-upstream#pull-in-core-from-pantheons-upstream).
+* Sites must be updated to the latest release of Drupal core (currently 7.101).
+<br/>
+<br/>
+
+To verify your current Drupal core version, you can check the [Status report tab](/drupal-launch-check#status-report) in the Pantheon site dashboard or you can use Terminus: 
+
+```bash{promptUser: user}
+terminus drush <site>.<env> status
+```
+
+To verify your current upstream, you can go to the Pantheon site dashboard and click **Settings**, then **About Site** or you can use Terminus: 
+
+```bash{promptUser: user}
+terminus site:info <site>
+```
+
+If you are not already using the Pantheon Drupal 7 upstream, you can [switch the site upstream via Terminus](/terminus/examples#switch-upstreams).
```

Comment:
If folks run this command without ensuring that their repository is aligned with the drops-7 repository, then the next dashboard update command could result in merge conflicts (best case) or destructive changes to the site.